development/build/index.js

```diff
@@ -114,6 +114,7 @@ async function defineAndRunBuildTasks() {
       'WeakSet',
       'Event',
       'Image', // Used by browser to generate notifications
+      'fetch', // Used by browser to generate notifications
```

Comment:
remove this line once https://github.com/MetaMask/metamask-extension/pull/24631#discussion_r1610277269 is fixed

Reply:
ACTUALLY - applying the idea behind this PR to the `Image` exception as well could be very beneficial, as Image is also a very strong capability